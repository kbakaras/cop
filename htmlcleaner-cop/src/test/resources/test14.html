<html xmlns:dc="http://purl.org/dc/elements/1.1/">
<body>
<dc:title>A test of mixing XML and HTML</dc:title>
</body></html>