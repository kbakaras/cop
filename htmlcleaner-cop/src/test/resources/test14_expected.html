<html xmlns:dc="http://purl.org/dc/elements/1.1/">
<head />
<body>

<dc:title>A test of mixing XML and HTML</dc:title>
</body></html>